module/os/linux/zfs/zvol_os.c

```diff
@@ -791,13 +813,21 @@ zvol_open(struct block_device *bdev, fmode_t flag)
 		rw_exit(&zv->zv_suspend_lock);
 
 	if (error == 0)
+#ifdef HAVE_BLK_MODE_T
+		disk_check_media_change(disk);
+#else
 		zfs_check_media_change(bdev);
+#endif
 
 	return (error);
 }
 
 static void
+#ifdef HAVE_BLOCK_DEVICE_OPERATIONS_RELEASE_VOID_1ARG
+zvol_release(struct gendisk *disk)
+#else
 zvol_release(struct gendisk *disk, fmode_t mode)
```

Comment:
We should probably annotate `mode` as unused below.

```suggestion
zvol_release(struct gendisk *disk, fmode_t unused)
{
	(void) unused;
	...
```
```